### DISCUSSION TITLE
What does it actually look like when it's working?

### DISCUSSION THREAD

**Discussion by 256shadesofgrey** [AsherGlick/Burrito#244] [Q&A] [answered]
I managed to get burrito to run, and seemingly communicate with the game client correctly. By using steamtinkerlaunch I got the burrito link to start with the game, and when I logged in, I saw that the burrito link window was actually showing some data, including my character name, so it's definitely doing something. And when I started the burrito.x86_64 in the console, it was also printing some information that looked like it was communicating correctly, like saying that it's loading a new map when I changed zones.

However I do not see anything on the screen in the game.

The website has no screenshots, and the "instructions" section says nothing beyond getting the application started. Is it supposed to show some icon like blish? Or does it only show markers after downloading them? And if there is no GUI to manage the markers, where do I put them?

Either something is not working on my system, or there needs to be a lot more documentation.

**Comment by AsherGlick** [comment, marked as answer]
Screenshots or a video would probably be a good addition to some documentation somewhere, new user experience will be a focus after we get cross platform testing setup. There is supposed to be an icon like taco/blish when burrito is passing input through.
![Screenshot from 2023-12-16 21-19-56](https://github.com/AsherGlick/Burrito/assets/603989/9c470d5c-1508-483a-b234-d61a8bb3c5ac)
This may be obscured by the window manager when gw2 is not in focus or if you have multiple monitors depending on the window manager. Burrito has logic to automatically resize to the gw2 window size, but it does not yet snap to the window coordinates like an overlay truly should.
Once you click the icon you will get a red border around all of burrito to indicate the burrito overlay itself is taking all the focus from the game.
![Screenshot from 2023-12-16 21-27-29](https://github.com/AsherGlick/Burrito/assets/603989/1fa0ef57-5d63-4ce6-bb28-ca384d508cbb)
Hope this helps.

**Reply by 256shadesofgrey** [reply]
Thank you, this is exactly what I was looking for.
